spring-jdbc/src/main/java/io/crate/spring/jdbc/samples/dao/ImagesJDBCTemplateDao.java

```diff
@@ -0,0 +1,97 @@
+package io.crate.spring.jdbc.samples.dao;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.net.http.HttpRequest.BodyPublishers;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.dao.DataIntegrityViolationException;
+import org.springframework.http.HttpStatus;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.lang.NonNull;
+import org.springframework.stereotype.Component;
+
+import io.crate.spring.jdbc.samples.model.Image;
+
+@Component
+public class ImagesJDBCTemplateDao implements ImagesDao {
+
+    @Value("${cratesample.cratedb.blobEndpointURL:http://localhost:4200/_blobs/}")
```

Comment:
`http://localhost:4200` is it supposed to be hardcoded?

Reply:
it is only a prototype, so it think it is ok